app/models/user_favourite.rb

```diff
@@ -19,7 +19,7 @@ def apply_limit
     #
     # Delete non-persistent favorites > 20
     #
-    UserFavourite
+    UserFavourite.default_scoped
```

Comment:
@bharat619 not sure why we would need that

Reply:
@patrixr  This is to avoid the warning
```
Class level methods will no longer inherit scoping from `new` in Rails 6.1. To continue using the scoped relation, 
pass it into the block directly. To instead access the full set of models, as Rails 6.1 will, use `UserFavourite.default_scoped`
```